gnucash/import-export/aqb/assistant-ab-initial.c

```diff
@@ -682,7 +683,8 @@ static void
 clear_kvp_acc_cb(gpointer gnc_acc, gpointer ab_acc, gpointer user_data)
 {
     g_return_if_fail(gnc_acc);
-    /* Delete complete "hbci..." KVPs for GnuCash account */
+    /* Delete "online-id" and complete "hbci..." KVPs for GnuCash account */
+    gnc_import_delete_acc_online_id((Account *) gnc_acc);
```

Comment:
I don't see the point of the indirection. Why not just call `gnc_account_delete_map_entry(account, "online_id", NULL, NULL, FALSE);` here?

Reply:
Done. I have removed the whole commit.